### PR TITLE
Some more words

### PR DESCRIPTION
**Pull request by sebweb3r**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -382,6 +382,7 @@ accoriding->according
 accoring->according, occurring,
 accoringly->accordingly
 accorndingly->accordingly
+accorting->according
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:385]: Can you add accort as another typo too and the accordance and accorded variants.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7074,6 +7075,9 @@ corresponging->corresponding
 corresponing->corresponding
 corresponsding->corresponding
 corresponsing->corresponding
+correspont->correspond
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7078]: Can you add the correspondent and correspondence versions of these typos too (and the plural versions of both).

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -18388,7 +18392,7 @@ numebrs->numbers
 numer->number
 numeraotr->numerator
 numerbering->numbering
-numerial->numeral
+numerial->numeral, numerical,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:18395]: Nice find!

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -6913,6 +6917,7 @@ coordonated->coordinated
 coordonates->coordinates
 coorespond->correspond
 cooresponded->corresponded
+coorespondend->correspondent
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:6920]: This probably wants to be `coorespondent` doesn't it?

Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:6920]: yes and no. i added the other variant too.

**Pull request merged**